node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerOperationsImpl.java

```diff
@@ -162,6 +174,23 @@ public void removeContainer(final Container existingContainer) {
 
         logger.info("Deleting container " + containerName.asString());
         docker.deleteContainer(containerName);
+
+        if (docker.networkNATed()) {
+            logger.info("Delete iptables NAT rules for " + containerName.asString());
+            try {
+                InetAddress nodeInetAddress = environment.getInetAddressForHost(nodeSpec.hostname);
+                String ipv6Str = docker.getGlobalIPv6Address(containerName);
+                String drop = NATCommand.drop(nodeInetAddress, InetAddress.getByName(ipv6Str));
+                Pair<Integer, String> result = processExecuter.exec(drop);
+                if (result.getFirst() != 0) {
+                    // Might be because the one or two (out of three) rules where not present - debug log and ignore
+                    // Trailing rules will always be overridden by a new one due the fact that we insert
+                    logger.debug("Unable to drop NAT rule - error message: " + result.getSecond());
```

Comment:
Isn't this a warning?

Reply:
I made it debug because it is not always an incorrect behavior and when it is, it does not impact the routing logic. This is more a maintenance task to avoid ever growing iptables.